public/global.css

```diff
@@ -10,6 +10,7 @@ body {
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
   scrollbar-width: none;
+  font-feature-settings: "ss01", "ss02", "cv01";
```

Comment:
This had to go here, because chromium doesn't support `font-feature-settings` in `@font-face` declarations.